client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-stacked-features.tsx

```diff
@@ -0,0 +1,52 @@
+import { Gridicon } from '@automattic/components';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { preventWidows } from 'calypso/lib/formatting';
```

Comment:
Nit: can you import only the `TranslateResult` type?
```suggestion
import { useTranslate } from 'i18n-calypso';
import { FunctionComponent } from 'react';
import { preventWidows } from 'calypso/lib/formatting';
import type { TranslateResult } from `i18-calypso';
```